src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -58,4 +58,17 @@ describe('FakeCacheService', () => {
 
     expect(target.keyCount()).toBe(0);
   });
+
+  it('increments the value of a key', async () => {
+    const key = faker.string.alphanumeric();
+    const firstResult = await target.incrementAndGet(key, undefined);
```

Comment:
This initially confused me - it creates missing keys, then increments them. It matches the functionality of Redis though so I think it's fine. I would, however, adjust the description to reflect this and add a second test that has a pre-existing value in the cache, starting from a higher number.